contracts/token/ERC20/SafeERC20.sol

```diff
@@ -5,36 +5,78 @@ import "../../math/SafeMath.sol";
 
 /**
  * @title SafeERC20
- * @dev Wrappers around ERC20 operations that throw on failure.
+ * @dev Wrappers around ERC20 operations that throw on failure (when the token
+ * contract returns false). Tokens that return no value (and instead revert or
+ * throw on failure) are also supported, non-reverting calls are assumed to be
+ * successful.
  * To use this library you can add a `using SafeERC20 for ERC20;` statement to your contract,
  * which allows you to call the safe operations as `token.safeTransfer(...)`, etc.
  */
 library SafeERC20 {
     using SafeMath for uint256;
 
     function safeTransfer(IERC20 token, address to, uint256 value) internal {
-        require(token.transfer(to, value));
+        callAndAssertSuccess(token, abi.encodeWithSignature("transfer(address,uint256)", to, value));
```

Comment:
I prefer `abi.encodeWithSelector(token.transfer.selector, ...)`.

Reply:
Ohh that's much nicer, I had forgotten about that one.